plugins/modules/homebrew.py

```diff
@@ -415,9 +415,9 @@ def _current_package_is_installed(self):
         if self.force_formula:
             cmd.append("--formula")
         rc, out, err = self.module.run_command(cmd)
-        if err:
+        if rc != 0:
             self.failed = True
-            self.message = err.strip()
+            self.message = err.strip()  or ("Unknown failure with exit code %d" % rc)
```

Comment:
One superfluous space:
```suggestion
            self.message = err.strip() or ("Unknown failure with exit code %d" % rc)
```

Reply:
I've been a little rusty. I haven't contributed to Ansible for a long time now, so I didn't have my dev dully set up. I've configured everything now to be able to run local tests, so all should be good with the next commit.

Thanks for your help!